<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>man_hinh_bao_cao_nam</class>
 <widget class="QWidget" name="man_hinh_bao_cao_nam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">border: None;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-4</y>
        <width>951</width>
        <height>701</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: None;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="styleSheet">
          <string notr="true">border: None;</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>181</width>
            <height>501</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 3px solid rgb(0, 0, 0);
border-radius: 10px;
background-color: rgb(255, 255, 255);

border-color: rgb(0, 112, 192);</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="xuat">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>340</y>
             <width>121</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Xuất File</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/images/334_n.png</normaloff>:/images/334_n.png</iconset>
           </property>
          </widget>
          <widget class="QPushButton" name="thoat">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>400</y>
             <width>121</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Thoát</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/images/32_n.png</normaloff>:/images/32_n.png</iconset>
           </property>
          </widget>
          <widget class="QComboBox" name="nam_hoc">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>210</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid rgb(0, 0, 0);
</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>185</y>
             <width>121</width>
             <height>22</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:  rgb(253, 255, 255);</string>
           </property>
           <property name="text">
            <string>Năm học:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>141</width>
             <height>131</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(253, 255, 255);
border-color: rgb(253, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../resource.qrc">:/images/333691756_871094670849167_873075225703648657_n.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="tao">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>280</y>
             <width>121</width>
             <height>51</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(85, 170, 255);
border-color: rgb(0, 0, 0);</string>
           </property>
           <property name="text">
            <string>Tạo</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource.qrc">
             <normaloff>:/images/6_n.png</normaloff>:/images/6_n.png</iconset>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>0</y>
            <width>731</width>
            <height>501</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2.5px solid rgb(0, 112, 192);
border-color: rgb(0, 112, 192);
background-color: rgb(0, 112, 192);
border-radius: 10px
</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>50</y>
             <width>331</width>
             <height>61</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid rgb(0, 112, 192);
border-radius: 10px;
</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>121</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Tổng số đề thi:</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="tong_so_de_thi">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>10</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>331</width>
             <height>61</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid rgb(0, 112, 192);
border-radius: 10px;
</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <widget class="QLineEdit" name="tong_so_bai_cham">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>10</y>
              <width>111</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 2px solid rgb(0, 112, 192);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 0 8px;
selection-background-color: darkgray;
font-size: 16px;</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>151</width>
              <height>22</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border:  rgb(253, 255, 255);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Tổng số bài chấm: </string>
            </property>
           </widget>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>10</y>
             <width>261</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tahoma</family>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(253, 255, 255);
background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Tahoma'; font-size:12pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:16pt; font-weight:696;&quot;&gt;BÁO CÁO NĂM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="bao_cao_nam">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>131</y>
             <width>711</width>
             <height>361</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 2px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
border-color: rgb(0, 112, 192);</string>
           </property>
          </widget>
         </widget>
         <zorder>groupBox_7</zorder>
         <zorder>groupBox_3</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
